pkg/v1/tkg/test/tkgctl/tkgs/tkgs_workload_cluster_test.go

```diff
@@ -171,6 +245,17 @@ func createClusterClassBasedClusterTest(tkgctlClient tkgctl.TKGClient, deleteClu
 	}
 }
 
+// deleteClusterClassBasedCluster deletes workload cluster
+func deleteClusterClassBasedCluster(tkgctlClient tkgctl.TKGClient, deleteClusterOptions tkgctl.DeleteClustersOptions, clusterName, namespace string) {
+	if isClusterClassFeatureActivated {
+		By(fmt.Sprintf("deleting cluster class based workload cluster %v in namespace: %v", clusterName, namespace))
+		err = tkgctlClient.DeleteCluster(deleteClusterOptions)
+		Expect(err).To(BeNil())
+	} else {
+		// TODO: Delete cluster when feature is not activated
+	}
+}
+
 // createLegacyClusterTest creates and deletes (if created successfully) workload cluster
```

Comment:
?? "creates and deletes", if it does both name of function should be reflect that.